client/knowledge_base_search/spec/smoke_test.js

```diff
@@ -0,0 +1,38 @@
+import { spawn } from 'child_process';
+
+describe('Knowledge Base Search', () => {
+  let server;
+
+  before((done) => {
+    // start the python server
+    server = spawn('python', [ '../../server/python/server.py' ]);
```

Comment:
And this is one of the things I actually like about karma-webpack, you can use the `resolve` option in the config to avoid annoying directory traversal. Bummer.